src/Backend/Modules/Groups/Entity/GroupActionRight.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Backend\Modules\Groups\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the GroupActionRight Entity
+ *
+ * @author Mathias Dewelde <mathias@dewelde.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="GroupActionRight")
```

Comment:
We should always support backwards compatibility for at least one minor version.
So developers can adjust all their modules relying on these **old** table names.
Because we don't know what crazy shit developers have created using existing modules...